packages/coding-standards/tests/Unit/Sniffs/ForbiddenSuperGlobalSniff/wrong/wrong.php.inc

```diff
@@ -0,0 +1,3 @@
+<?php
+
+$password = $_POST['password'];
```

Comment:
also here, please rename files to more descriptive names like `post.php.inc`, `env.php.inc`